salt/modules/uboot.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+'''
+Interface to the U-BOOT environement variables
+
+(Parsing through fw_printenv/fw_setenv)
+
+External References
+-------------------
+| `U-BOOT http://www.denx.de/wiki/U-Boot`_
+
+'''
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+import salt.utils
+
+# Solve the Chicken and egg problem where grains need to run before any
+# of the modules are loaded and are generally available for any usage.
+import salt.modules.cmdmod
```

Comment:
I see where you're going with this but this is really starting to feel like a lot of overhead just to make a single shell call. We already have access to to the `cmd` module in the core grains. Why not just call what we need using that there? If we want to ship an execution module to wrap calls to `fw_*`, that's fine, but I don't think that we should be coupling together the core grains and an execution module together in this way.

Please let me know what you think. :]